datasets/readme.md

```diff
@@ -0,0 +1,9 @@
+## `freesolv`
+`freesolv.sdf` contains a snapshot of the FreeSolv database of hydration free energies curated by David Mobley, Peter Guthrie, and colleagues here: https://github.com/MobleyLab/FreeSolv
+
+TODO(YTZ) : note which version of FreeSolv is stored, and if any other filtering or processing has been applied 
```

Comment:
```
15:35 $ git log
commit ff0961a3177863c8002e8487ff9718c513974138 (HEAD -> master, origin/master, origin/HEAD)
Merge: 9d8a809 e332a44
Author: David L. Mobley <l5a9i0q4q3p4w2q5@mobleylab.slack.com>
Date:   Fri Jan 26 10:38:04 2018 -0800
```

Reply:
Resolved in https://github.com/proteneer/timemachine/pull/331/commits/0b3617ad14baf0486020c94c128f88fabbcee6e3